content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -646,7 +646,8 @@ You can override the default shell settings in the runner's operating system usi
 | All | `python` | Executes the python command. | `python {0}` |
 | Linux / macOS | `sh` | The fallback behavior for non-Windows platforms if no shell is provided and `bash` is not found in the path. | `sh -e {0}` |
 | Windows | `cmd` | {% data variables.product.prodname_dotcom %} appends the extension `.cmd` to your script name and substitutes for `{0}`. | `%ComSpec% /D /E:ON /V:OFF /S /C "CALL "{0}""`. |
-| Windows | `powershell` | This is the default shell used on Windows. The Desktop PowerShell. {% data variables.product.prodname_dotcom %} appends the extension `.ps1` to your script name. | `powershell -command ". '{0}'"`. |
+| Windows | `pwsh` | This is the default shell used on Windows. The PowerShell Core. {% data variables.product.prodname_dotcom %} appends the extension `.ps1` to your script name. | `pwsh -command ". '{0}'"`. |
```

Comment:
This is absolutely correct for Hosted runners but I have small note about self-hosted:
If self-hosted Windows runner doesn't have `PowerShell Core` installed, default shell falls back to `PowerShell Desktop`.
Related code: https://github.com/actions/runner/blob/main/src/Runner.Worker/Handlers/ScriptHandler.cs#L87

@martin389, do you think it should be mentioned?

Reply:
Thanks @maxim-lobanov, makes sense to me 👍  I'll add a note to mention this.